src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseMultiSignMessage.cs

```diff
@@ -33,6 +41,37 @@ internal CoseMultiSignMessage(CoseHeaderMap protectedHeader, CoseHeaderMap unpro
             _signatures = signatures;
         }
 
+        /// <summary>
+        /// Signs the specified content and encodes it as a COSE_Sign message with detached content.
+        /// </summary>
+        /// <param name="detachedContent">The content to sign.</param>
+        /// <param name="signer">The signer information used to sign <paramref name="detachedContent"/>.</param>
+        /// <param name="protectedHeaders">The protected header parameters to append to the message's content layer.</param>
+        /// <param name="unprotectedHeaders">The unprotected header parameters to append to the message's content layer.</param>
+        /// <param name="associatedData">The extra data associated with the signature, which must also be provided during verification.</param>
+        /// <returns>The encoded message.</returns>
+        /// <exception cref="ArgumentNullException">
+        ///   <para>
+        ///     <paramref name="detachedContent"/> is <see langword="null"/>.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     <paramref name="signer"/> is <see langword="null"/>.
+        ///   </para>
+        /// </exception>
```

Comment:
There's a shorthand for ArgumentNullException (or some other situation where everything throws for the same reason just varies on the parameter name):

```suggestion
        /// <exception cref="ArgumentNullException">
        ///   <paramref name="detachedContent"/> or <paramref name="signer"/> is <see langword="null"/>.
        /// </exception>
```

https://github.com/dotnet/dotnet-api-docs/wiki/Exceptions

> If an exception can occur under the same condition but with > multiple arguments, you can describe the condition in one sentence. For example:
> 
> T:System.ArgumentNullException: The _type_, _target_, or _method_ parameter is a null >reference (**Nothing** in Visual Basic).

(of course, now we just say `is <see langword="null" />`)